src/KUSignalLib/communications/communications.py

```diff
@@ -82,4 +82,25 @@ def srrc(alpha, m, length):
             pulse.append(1.0)  # Handle division by zero case
         else:
             pulse.append(num/den)
-    return pulse
\ No newline at end of file
+    return pulse
+
+def bin_to_symbol(sequence, constellation = None):
```

Comment:
i feel like this is very specific to each modulation type and maybe something that the user can add in their own script instead of relying on a library function call....